.github/workflows/precommit.yml

```diff
@@ -102,15 +102,15 @@ jobs:
       - name: Install tox
         run: python -m pip install tox
       - if: ${{ matrix.python-version == '3.8' && contains('macos', matrix.os.download_name) }}
-        name: Run tests (including doctest Tests)
-        run: python -m tox -e "py${PYTHON_VERSION/\./}-udf"
+        name: Run doctests
+        run: python -m tox -e "py${PYTHON_VERSION/\./}-doctest-notudf"
```

Comment:
Could remove the `/\./` here

Reply:
what is the function of `/\./`

Reply:
`/\./` means nothing is specified, which falls under `!unit-!integ-!doctest: SNOWFLAKE_TEST_TYPE = (unit or integ or doctest)` and ends up running all tests. 
Here I specify `doctest` explicitly to only run doctests, so the `/\./`  is meaningless.